core/eolearn/core/constants.py

```diff
@@ -10,15 +10,46 @@
 This source code is licensed under the MIT license found in the LICENSE
 file in the root directory of this source tree.
 """
-from enum import Enum
+import warnings
+from enum import Enum, EnumMeta
 from typing import Optional
 
 from sentinelhub import BBox, MimeType
 
+from .exceptions import EODeprecationWarning
+
 TIMESTAMP_COLUMN = "TIMESTAMP"
 
 
-class FeatureType(Enum):
+def _warn_and_adjust(name: str) -> str:
+    # since we stick with `UPPER` for attributes and `lower` for values, we include both to reuse function
+    deprecation_msg = None
+    if name in ("TIMESTAMP", "timestamp"):
+        deprecation_msg = (
+            "`FeatureType.TIMESTAMP` has been renamed to `FeatureType.TIMESTAMPS` and will be removed in a future"
+            " version. Please switch to using `FeatureType.TIMESTAMPS`."
+        )
+        name = "TIMESTAMPS" if name == "TIMESTAMP" else "timestamps"
+
+    if deprecation_msg:
+        warnings.warn(deprecation_msg, category=EODeprecationWarning, stacklevel=3)
```

Comment:
as discussed in person, do we skip the deprecation warning and just autocorrect? Since it will be removed soon anyway

Reply:
Yes, we can skip the deprecation and just do the inclusion.

Reply:
warning removed, but I left the functionality there  for later (had to add a mypy ignore statement because the code was never reached)